pkg/ansibleapp/types.go

```diff
@@ -2,32 +2,45 @@ package ansibleapp
 
 import (
 	"encoding/json"
+	"github.com/pborman/uuid"
 )
 
+type Parameters map[string]interface{}
```

Comment:
Reading this blog, `map[string]interface{}` is the generic type go uses for JSON objects in that the entire object should still be accessible by normal golang means: https://blog.golang.org/json-and-go

"The json package uses map[string]interface{} and []interface{} values to store arbitrary JSON objects and arrays; it will happily unmarshal any valid JSON blob into a plain interface{} value."

This type allows us to access stuff inside of it, as opposed to an opaque string blob.